.ci.yaml

```diff
@@ -156,6 +156,19 @@ targets:
       emulator_arch: "x64"
     timeout: 60
 
+  - name: Linux Fuchsia FEMU (cso)
+    recipe: engine/femu_test
+    bringup: true
+    properties:
+      add_recipes_cq: "true"
+      build_fuchsia: "true"
+      fuchsia_ctl_version: version:0.0.27
+      # ensure files from pre-production Fuchsia SDK tests are purged from cache
+      clobber: "true"
+      emulator_arch: "x64"
+      enable_cso: "true"
+    timeout: 60
```

Comment:
Would 60 mins be enough? usually checking out all the repositories from a clean cache takes ~14 mins.

Reply:
Increased to 90